src/lib/utils/sqlite_wrapper.cpp

```diff
@@ -54,7 +54,6 @@ void SQLiteWrapper::create_table(const Table& table, const std::string& table_na
         column_types.emplace_back("TEXT");
         break;
       case DataType::Null:
-      case DataType::Bool:
```

Comment:
Is bool a jit-related type?

Reply:
Yes. See the deleted comment in types.hpp